README.md

```diff
@@ -70,6 +70,7 @@ The Nsq::Producer constructor takes the following options:
 | `nsqd`        | Host and port of the nsqd instance     | '127.0.0.1:4150'   |
 | `nsqlookupd`  | Use lookupd to automatically discover nsqds |               |
 | `ssl_context` | Optional keys and certificates for TLS connections |        |
+| `tls_v1`      | Flag for tls v1 connections (default false) |               |
```

Comment:
Good points, both resolved in https://github.com/wistia/nsq-ruby/pull/26/commits/923dd7b72e489e4d554e48f2820dd264b06594c6